#0x0000: ok, 0xFFFF: offline
uint16 status
